ipi/pes/ase.py

```diff
@@ -81,15 +82,27 @@ def __call__(self, cell, pos):
         structure.calc = self.ase_calculator
 
         # Do the actual calculation
-        properties = structure.get_properties(self.capabilities)
+        self.ase_calculator.calculate(structure)
```

Comment:
Depending on the calculator, one might still have to explicitly pass the requested properties to compute; so we might want to keep the previous way of calling this, but adding more stuff to `self.capabilities`, according to user request.